### PR TITLE
Feat/child routing

### PR DESCRIPTION
**Pull request by Aleksa013**
#### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 💡 Background and solution
image

_Describe the big picture of your changes here_
add routing into admin-page
#### 🎯 Related issues

_Links to related issues_

### CODE REVIEW

**Review by Arthurchkeee — APPROVED** [review]
Good job! 💯

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -32,7 +35,25 @@ const routes: Routes = [
...
+        path: 'carriages',
+        component: CarriageComponent,
+      },
+      {
+        path: 'routes',
+        component: RoutesComponent,
+      },
+      {
+        path: 'stations',
+        component: CarriageComponent,
```
Comment (NataRep) [inline src/app/app-routing.module.ts:53]: Am I understanding correctly that these are sample components for pages and then we'll post the real ones here?

File: src/app/pages/admin-page/admin-page.component.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @media (max-width: 500px) {
+      width: 100%;
+      flex-direction: row;
+      button {
+        width: 30%;
+      }
+    }
+    .non-active {
+      background-color: var(--grey-light);
+    }
```
Comment (NataRep) [inline src/app/pages/admin-page/admin-page.component.scss:30]: please let's make the current link with these styles:
```
background-color: var(--blue-extra-light);
    color: #fff;
```

File: src/app/pages/admin-page/admin-page.component.html [hunk 3]
```diff
@@ -1 +1,27 @@
...
+  <button
+    mat-button
+    routerLink="carriages"
+    (click)="makeActive('carriages')"
+    [ngClass]="activeButtons === 'carriages' ? 'active' : 'non-active'"
+  >
+    Carriages
+  </button>
+</div>
+<router-outlet></router-outlet>
```
Comment (NataRep) [inline src/app/pages/admin-page/admin-page.component.html]: let's wrap in a Section tag to add positioning
```
<section style=''content-wrapper>
<router-outlet></router-outlet>
</section>
```
```
.content-wrapper
{
    padding-left: 2rem;
    @media (max-width: 500px) {
       padding-left: 0;
       padding-top: 2rem;
  }
}
```

**Review by NataRep — CHANGES_REQUESTED** [review]
The logic is fine! I've added a couple of comments on how you can adjust some styles

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Aleksa013) [reply src/app/app-routing.module.ts:53]: Your component was ready when I started these task. I used it. Routes component will full by me and Artur will add Stations Component

**Review by NataRep — APPROVED** [review]

**Review by mogitrash — APPROVED** [review]
💯

**Pull request merged**